### PR TITLE
Update README to match formatted geojson

### PR DESCRIPTION
**Pull request by briannapoulos**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -65,20 +65,29 @@ The catchment area boundaries, uniquely generated by Crosscut, with all correspo
...
-Google Buildings 70+% Confidence| The total count of buildings estimated to be within the catchment area, based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 70% or greater confidence that they are buildings
-Google Buildings 80+% Confidence| The total count of buildings estimated to be within the catchment area, based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 80% or greater confidence that they are buildings
-Average Building Footprint (m2) | Average square meters of the land area covered by each building in the catchment area based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 80% or greater confidence that they are buildings
+| Name | The name associated with the catchment area |
+| Site Location | The latitude and longitude of the site associated with the catchment area (if applicable) |
+| Admin 0 | The admin 0 name associated with the catchment area |
+| Admin 1 | The admin 1 name(s) associated with the catchment area |
```
Comment (mckinnoj) [inline profile/README.md:71]: Would add: "Multiple names will be given if the catchment area crosses multiple administrative boundaries."

File: profile/README.md [hunk 2]
```diff
@@ -65,20 +65,29 @@ The catchment area boundaries, uniquely generated by Crosscut, with all correspo
...
+| Admin 2 | The admin 2 name(s) associated with the catchment area (if applicable) |
```
Comment (mckinnoj) [inline profile/README.md:72]: Would add: "Multiple names will be given if the catchment area crosses multiple administrative boundaries."

File: profile/README.md [hunk 3]
```diff
@@ -65,20 +65,29 @@ The catchment area boundaries, uniquely generated by Crosscut, with all correspo
...
+| Admin 3 | The admin 3 name(s) associated with the catchment area (if applicable) |
```
Comment (mckinnoj) [inline profile/README.md:73]: Would add: "Multiple names will be given if the catchment area crosses multiple administrative boundaries."

File: profile/README.md [hunk 4]
```diff
@@ -65,20 +65,29 @@ The catchment area boundaries, uniquely generated by Crosscut, with all correspo
...
+| Admin 4 | The admin 4 name(s) associated with the catchment area (if applicable) |
```
Comment (mckinnoj) [inline profile/README.md:74]: Would add: "Multiple names will be given if the catchment area crosses multiple administrative boundaries."

**Review by mckinnoj — APPROVED** [review]
I left a handful of line comments (assuming I'm understanding the Github interface), but looks great!

File: profile/README.md (hunk 3 above)
Comment (briannapoulos) [reply profile/README.md]: 370323fa83627e9e3a6f66a1f4e1439edec3232c

File: profile/README.md (hunk 2 above)
Comment (briannapoulos) [reply profile/README.md]: 370323fa83627e9e3a6f66a1f4e1439edec3232c

File: profile/README.md (hunk 1 above)
Comment (briannapoulos) [reply profile/README.md]: 370323fa83627e9e3a6f66a1f4e1439edec3232c

**Pull request merged**